coffee_core/src/coffee.rs

```diff
@@ -315,7 +315,11 @@ impl PluginManager for CoffeeManager {
     }
 
     async fn add_remote(&mut self, name: &str, url: &str) -> Result<(), CoffeeError> {
-        self.remote_sync().await?;
+        // FIXME: we should allow some error here like
```

Comment:
Maybe we can call the function only if the directory exists?
```
if Path::new(&format!("{}/repositories/", self.config.root_path)).exists() {
            self.remote_sync().await?;
}
```

Reply:
No this should be a generic step; I need to think a little bit more.

The nurse command is broken, and we should improve it and not spread this hack across the code base imho.